src/microbe_stage/MicrobeHUD.cs

```diff
@@ -170,6 +170,10 @@ public class MicrobeHUD : Node
     private GridContainer environmentPanelBarContainer;
     private Panel compoundsPanel;
 
+    // Store these statefully for after player death
+    private float maxHP;
+    private float maxATP;
```

Comment:
These should default to value of 1 so that there aren't any divide by zero errors if the player object isn't set immediately when the game starts.

Reply:
I don't see any places where they are actually used in a denominator. Line 799 is string concatenation, not division.

Reply:
Godot does that internally, at least it is not happy to have a bar max value be 0 and it will print errors.